README.md

```diff
@@ -1,9 +1,10 @@
 # lua-func
 
 The Lua interpreter split in a way such that each function is in it's own file. Except for a few
-functions that were very hard to split up, for example the `lundump.c` file. 
+functions that were very hard to split up, for example the `lundump.c` file. Doing that would not
```

Comment:
What does "doing that would not give additional CHERI security guarantees" mean?

Reply:
Is it more understandable now?